PROJECT

```diff
@@ -44,4 +43,13 @@ resources:
   kind: RsyncDataMovement
   path: github.com/NearNodeFlash/nnf-dm/api/v1alpha1
   version: v1alpha1
+- api:
```

Comment:
Rsync references are still in this file: `RsyncTemplate`, `RsyncNodeDataMovement`, `RsyncDataMovement`